src/components/sandbox-menu/sandbox-menu.tsx

```diff
@@ -88,7 +98,7 @@ function saveConfig(baseUrl: string, config: Record<string, unknown>) {
   getStore().set(CONFIGURATION_TOUCHED_REDUCER_ATOM, {
     type: 'set-multiple',
     value: Object.fromEntries(
-      Object.keys(finalConfigEntries).map((key) => [key, true]),
+      Object.keys(userConfigEntries).map((key) => [key, true]),
```

Comment:
So touched is only for the user values?

Reply:
Correct. It is a bit more difficult because it is a custom menu. In the simple case, let's say a Switch, the touched is set to true if the user changed its value (regardless of what the value is). In the Popover where you can trash your customisation for an option, this value is set back to false:
 
https://github.com/UnofficialCrusaderPatch/UCP3-GUI/blob/c6b5e70c4922c8cf04ae6c2c3986adc3c0ec58c8/src/components/ucp-tabs/config-editor/ui-elements/ui-factory/popover/ConfigPopover.tsx#L166-L169

I think in general, all the keys in touched that are true, are also in the user config. So all the things the user changed the value of in the custom menu should have `touched` set to `true`. Is this still the case with the code I wrote? Or does the custom menu set more things other than is represented in `userConfigEntries`?

Reply:
Nah, that is fine.